tsdb/docs/format/memory_snapshot.md

```diff
@@ -3,6 +3,11 @@
 Memory snapshot uses the WAL package and writes each series as a WAL record.
 Below are the formats of the individual records.
 
+The order of records in the snapshot is always:
+1. Starts with series records, one per series, in an unsorted fashion.
+2. After all series are done, we write a tombstone record containing all the tombstones.
+3. At the end, we write one or more exemplar records while batching up the exemplars in each record. The order of exemplars is same as it appears in the circular buffer in the memory.
+
```

Comment:
Since this is a memory snapshot, we write the chunk directly along with series. A series record is everything that is in the memory for the series. The format [here](https://github.com/prometheus/prometheus/blob/main/tsdb/docs/format/memory_snapshot.md#series-records) will make it more clear.